Sources/PackageGraph/PackageGraph.swift

```diff
@@ -33,8 +33,12 @@ public struct PackageGraph {
         return rootPackages.flatMap{$0.modules}.contains(module)
     }
 
+    /// The errors encountered while loading the package graph.
+    public let errors: [Swift.Error]
```

Comment:
I filed `<rdar://problem/30628202> Report discovered errors via delegate` for tracking early error reporting work.